src/main/java/com/jcraft/jsch/bc/CAST128CBC.java

```diff
@@ -0,0 +1,91 @@
+/* -*-mode:java; c-basic-offset:2; indent-tabs-mode:nil -*- */
+/*
+Copyright (c) 2005-2018 ymnk, JCraft,Inc. All rights reserved.
```

Comment:
This file is just added. Is it copyrighted by JCraft? Question applies to other freshly added files as well.

From the blog post http://www.matez.de/index.php/2020/06/22/the-future-of-jsch-without-ssh-rsa/ I understood, that JCraft is no longer interested in developing this software. So why new files keep this notice unchanged?

Reply:
It was copied from an existing file authored by JCraft and then lightly modified. Therefore I believe any original copyright notices should remain intact.